Common/HttpWebRequestWrapperLib/HttpWebRequestWrapper.cs

```diff
@@ -0,0 +1,137 @@
+        {
+            if (string.IsNullOrEmpty(url))
+            {
+                return "Url is null or empty";
+            }
+            if (!(parameters is null) && parameters.Count > 0)
+            {
+                url = getUrlWithParams(url, parameters);
+            }
+            httpWebRequest.Method = "GET";
+            HttpWebRequest.Create(url);
+
+            httpWebResponse = (HttpWebResponse)httpWebRequest.GetResponse();
+            using Stream responseStream = httpWebResponse.GetResponseStream();
+            using StreamReader streamReader = new StreamReader(responseStream, Encoding.UTF8);
+            var result = streamReader.ReadToEnd();
+            return result;
+        }
+
+        public string Put(string url, Dictionary<string, string> parameters, string body)
+        {
+            if (string.IsNullOrEmpty(url))
+            {
+                return "Url is null or empty";
+            }
+            if (string.IsNullOrEmpty(body))
+            {
+                return "Data in body is null or empty";
+            }
+            if (!(parameters is null) && parameters.Count > 0)
+            {
+                url = getUrlWithParams(url, parameters);
+            }
+
+            httpWebRequest.Method = "PUT";
+            HttpWebRequest.Create(url);
+
+            using Stream requestStream = httpWebRequest.GetRequestStream();
+            using StreamWriter streamWriter = new StreamWriter(requestStream, Encoding.UTF8);
+            streamWriter.WriteLine(body);
+
+            httpWebResponse = (HttpWebResponse)httpWebRequest.GetResponse();
+            using Stream responseStream = httpWebResponse.GetResponseStream();
+            using StreamReader streamReader = new StreamReader(responseStream, Encoding.UTF8);
+            var result = streamReader.ReadToEnd();
+            return result;
+        }
+
+        public string Post(string url, Dictionary<string, string> parameters, string body)
+        {
+            if (string.IsNullOrEmpty(url))
+            {
+                return "Url is null or empty";
+            }
+            if (string.IsNullOrEmpty(body))
+            {
+                return "Data in body is null or empty";
+            }
+            if (!(parameters is null) && parameters.Count > 0)
+            {
+                url = getUrlWithParams(url, parameters);
+            }
+
+            httpWebRequest.Method = "POST";
+            HttpWebRequest.Create(url);
+
+            using Stream requestStream = httpWebRequest.GetRequestStream();
+            using StreamWriter streamWriter = new StreamWriter(requestStream, Encoding.UTF8);
+            streamWriter.WriteLine(body);
+
+            httpWebResponse = (HttpWebResponse)httpWebRequest.GetResponse();
+            using Stream responseStream = httpWebResponse.GetResponseStream();
+            using StreamReader streamReader = new StreamReader(responseStream, Encoding.UTF8);
```

Comment:
=)

Reply:
Sorry for smiles, but copypasting became soooo boring at that point xd

Reply:
fixed